src/workspace/workspace_navigation.rs

```diff
@@ -25,6 +27,14 @@ impl WorkspaceNavigation {
             Err(ParseError::new(format!("No argument was passed")))
         }
     }
+
+    pub fn is_create_if_not_exists(&self) -> bool {
```

Comment:
great idea, we could also add this to the config

Do you not like `New` ? 
I mean this would be rather descriptive and simple, but we could leave it at `NextFree` if you like
I would then change this in the manpages

Reply:
I don't know, to be honest i neither like new nor nextfree...
Im not to sure, if I would add this to the config as extra attribute like execute ?:)

Reply:
yeah exactly. You need to give the config struct a new field and define a default function and value. thats pretty much it

Reply:
As for the naming, i dont have a better Idea then `new` tbh, sorry

Reply:
i like next free better... do you want me to change the writing ??

>yeah exactly. You need to give the config struct a new field and define a default function and value. thats pretty much it

Im not sure that is something the user needs to configure ?? :)